Gopkg.toml

```diff
@@ -25,12 +25,17 @@
 #   unused-packages = true
 
 required = [
+    "github.com/securego/gosec/cmd/gosec",
     "github.com/golang/protobuf/protoc-gen-go",
     "github.com/gogo/protobuf/protoc-gen-gogo",
     "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway",
     "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
 ]
 
+[[constraint]]
+  name = "github.com/securego/gosec"
+  version = "1.3.0"
```

Comment:
dep will consider this as "^1.3.0" by default. To pin, use "=1.3.0"

Reply:
I think it's fine to upgrade when they do new releases, the issue is that today we were pulling master every time. Hopefully numbered releases will be more stable.

Reply:
I also expect that this will be removed by `dep` the next time someone runs update because its not required by the project directly. I suggest adding this as a `required` package. 
See https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md#required